templates/article.liquid

```diff
@@ -1,9 +1,25 @@
 <section id="article">
-  <div class="grid-row">
-    <div class="grid-item item-s-12 item-m-6">
-      <h2>{{ article.title }}</h2>
-      <p>{{ article.published_at | date: '%d %B %Y' }}</p>
-      {{ article.content }}
+  <div class="container">
+    <div class="grid-row full-height">
+      <div class="grid-item item-s-12 item-m-6 border-right-half grid-row no-gutter">
```

Comment:
I feel like this works better with the m size bigger and l set to 6.

Reply:
Also because in the archive view the posts are actually set always to s-10 there is a strange jump in padding between the archive and the single. Not sure if there is an easy solution to that.